portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,31 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<body>
+    <ul class="topnav">
+    <li><a href="index.html">Isabel Mitre</a></li>
+    <li class="right"><a href="pages/about.html">ABOUT ME</a></li>
+    <li class="right"><a href="pages/projects.html">PROJECTS</a></li>
+    <li class="right"><a href="pages/gallery.html">GALLERY</a></li>
+    <li class="right"><a href="index.html">HOME</a></li>
+    </ul>
+    <div class="content">
+    <img id="profile" src="images/pretty2(1).jpg"/>
+    <div id="text">
+        <h1>I am Isabel Mitre</h1>
+        <h2>Hi! Glad you're here. Here's a little about me...</h2>
+        <p>I am a student at the University of Florida and currently have an internship 
+            at Google! (Thanks Google!) </p>
+        <p>I am excited to learn, BLAH, BLAH, BLAH. You can learn more about me 
+            <a href="pages/about.html">here</a>. </p>
+        <p>Thanks for stopping by! Feel free to say hi on 
+            <a href="https://www.instagram.com/1sa.mitre/?hl=en">Instagram</a> or 
+            <a href="https://www.linkedin.com/in/isabel-mitre/">LinkedIn</a> and feel free to check out my 
+            <a href="https://github.com/isamitre">GitHub! </a></p>
+      </div>
+        <!--<button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>
+        -->
```

Comment:
Remove this?

Reply:
I didn't want to not have it later if I wanted it, but I don't so I will